src/Features/CSharp/Portable/Completion/CSharpCompletionService.cs

```diff
@@ -22,7 +22,6 @@ internal class CSharpCompletionService : AbstractCompletionService
             {
                 new AttributeNamedParameterCompletionProvider(),
                 new NamedParameterCompletionProvider(),
-                new SpeculativeTCompletionProvider(),
```

Comment:
Since we're specifying an order here, is there a change in behavior by removing the first one rather than the second one?